### ISSUE TITLE
[ICC] LK - Valk didnt spawn on p2

### ISSUE THREAD

**Issue by Derolana** [dalaranwow/dalaran-wow#5221]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Valkyr didnt spawn on p2 
**Expected behaviour**: Tell us what should happen instead
A valk should spawn and grab someone.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**
https://streamable.com/ji7ga
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by JohnADC** [comment]
Are you sure the DBM timer is correct for that exact valkyr? We did see some diffrences in DBM timers with real timers on this server and i belive one of the valkyrs doesnt show as on timer(a few defiles off timer aswell), also a few other bosses abilities(like sindragosa second blistering in phase2 for example) arent the same.

**Comment by Derolana** [comment]
ill post a longer vesrion of that video
https://streamable.com/3sb7y  there is defile after defile 

and the first lk10 video i made https://youtu.be/vXSTePm2hy4?t=3m20s to 7:05 every valk spawned on time

**Comment by alexuktc** [comment]
Defile timers are not exact, because they are not meant to be. The cooldown of the skill is off, meaning that it can be cast once the cooldown is off and then another timer runs that has a random variable for when it is actually cast. This is intended, as knowing exactly when Defile is going off would trivialize the mechanic.

**Comment by Derolana** [comment]
by posting this i meant we should get that valkyr between those 2 defiles, not for exact timing on defile. atleast i saw this for first time. so it could be my fault idk

**Comment by alexuktc** [comment]
I do not have the exact cooldowns for both skills, but if I understand you correctly you are right. Defile's cooldown is a bit longer than the Valks', but normally you should not have 2 Valks without a Defile nor the other way around. 

Defile and Valks being cast together is a possibility and happened in retail as well. Guides even mention it as the biggest challenged of p.2 .

**Comment by Derolana** [comment]
Also we did bane today, i have 2 videos of valk not spawning again before transition after p2. now im really not sure if this is a bug or just they not spawn

**Comment by Lothloryen** [comment]
Hello there,

Thank you very much for your report, it is really appreciated. I couldn't reproduce this myself by any means, however I shall deploy some checks and protection for it to Algalon.
